assets/govuk_frontend_toolkit/stylesheets/_colours.scss

```diff
@@ -103,8 +103,10 @@ $panel-colour: $grey-3;           // Related links panel, page footer etc.
 $canvas-colour: $grey-4;          // Page background
 $highlight-colour: $grey-4;       // Table stripes etc.
 $page-colour: $white;             // The page
-$alpha-colour: $pink;             // Alpha badges and banners
-$beta-colour: $orange;            // Beta badges and banners
+$discovery-colour: $govuk-blue;   // Discovery badges and banners
+$alpha-colour: $govuk-blue;       // Alpha badges and banners
+$beta-colour: $govuk-blue;        // Beta badges and banners
+$live-colour: $grass-green;       // Live badges and banners
```

Comment:
Same goes for `$live-colour`

Reply:
https://github.com/alphagov/govuk_elements/search?utf8=%E2%9C%93&q=live-colour&type=